vnext/ReactUWP/Modules/AppThemeModuleUwp.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <Modules/AppThemeModule.h>
+
+#include <winrt/Windows.UI.ViewManagement.h>
+
+#include <cxxreact/MessageQueueThread.h>
+
+#include <IReactInstance.h>
+
+namespace react { namespace uwp {
+
+class AppTheme : public react::windows::AppTheme
```

Comment:
>AppTheme [](start = 40, length = 8)

Is there any reason we need AppThemeModule in ReactWindowsCore/Modules, and then AppThemeModuleUwp?
I think AppThemeModuleUwp is enough

Reply:
This was covered in another comment. It's there in case win32 wants to do their own implementation.